config/pipeline.yaml

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Configuration for running the complete pipeline including modeling and evaluation.
+
+# Absolute paths to non-code resources.
+file_paths:
+  queries: "/home/jupyter/github_msba_capstone/msba-capstone-2020/queries"
+
+# Global config used by multiple pipeline steps. Also made available
+# as parameters to all SQL queries.
+global:
+  # Project and dataset configuration.
+  source_project_id: "bigquery-public-data"
+  destination_project_id: "ut-goog"
+  source_dataset: "cfpb_complaints"
+  destination_dataset: "cfpb_complaints"
+
+  # Clean data, minor refined versions of provided data.
```

Comment:
"provided data" -> "source data"

Reply:
I may be wrong about this. But I'm confused what "provided" data means. It's a bit confusing to tell what the "source_table" is. Should this maybe be grouped with "source_dataset" above?

Generally, add some comments, when necessary, to make the meaning of a config value clearer.

Reply:
Maybe the "source" config and the "pipeline" config should be the separate sections? That might be clearer.

Reply:
Or at least separate what's read in from the public data source vs. what's written into the user's project.

Reply:
I have separated the source and destination datasets and tables. I have also explained it briefly in the comments.